libs/atqr/domain/src/lib/challenge/challenge.entity.ts

```diff
@@ -29,33 +29,43 @@ export class Challenge {
   private _status?: ChallengeStatus;
 
   constructor(
-    private _goal: string,
-    private _supervisorName: string,
-    private _supervisorEmail: EmailAddress,
-    private _player: Player,
-    price: number,
-    deadline: Date,
-    private _paymentMethod?: PaymentMethodEntity,
-    _status: ChallengeStatus = ChallengeStatus.Ongoing,
-    private _supervisorStatus: SupervisorEnum = SupervisorEnum.notInvited
+    private _goalId: GoalId,
```

Comment:
where is the challengeId property?

Reply:
it is a prop of the class and it is in disl definition

Reply:
I meant it's parameter. You didn't add it.

Reply:
@albnunes

Reply:
" challengeId: ChallengeId " prop is in Challenge's entity proprieties , and it is set with "private" , so we don't have to call it again inside the constructor